dataland-frontend/src/components/pages/ChooseFrameworkForDataUpload.vue

```diff
@@ -137,60 +124,127 @@ export default defineComponent({
     },
   },
 
-  watch: {},
   methods: {
     /**
-     *  Sorts a list of data meta information by their uploading time
+     * Function building a unified for subtitle for a framework type
+     *
+     * @param dataTypeTitle the type of the framework (humanized name)
+     * @returns a unified subtitle
+     */
+    buildSubtitle(dataTypeTitle: string): string {
+      return `Overview of all existing ${dataTypeTitle} datasets for this company.`;
+    },
+
+    // TODO test this in unit tests
```

Comment:
open Todos in this file

Reply:
@fsoffel  Have you written a component test for this page or its functions? If yes, we can remove all "TODO test this in unit tests" tags here

Reply:
I think the TODO is gone now since Florian wrote tests for this page